wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1307,13 +1292,6 @@ extension ZMUserSession: SyncAgentDelegate {
                 WireLogger.mls.warn("`qualifiedClientID` is missing for selfClient")
             }
 
-            if !isRecovering, mlsFeature.isEnabled {
```

Comment:
this is now replaced by any item generated by the CommitPendingProposalGenerator